### ISSUE TITLE
Should it ask for birth date immediately upon sign-up? 

### ISSUE THREAD

**Issue by mwestwick** [IserveU/IserveU#228]
not sure if that was the intention, but it would seem to make the most sense, in my view. In my case, had to enter it into my profile after sign up.

**Comment by psaunders** [comment]
Probably, it's a good way to prevent duplicate accounts from being spawned.

Send an email to a person we have on record with matching Name/DOB and say "Is this you?" and they can delete the duplicate account.

**Comment by mwestwick** [comment]
Yeah, it didn't ask for mine off the top. Had to go to my profile after to set that up.

**Comment by psaunders** [comment]
I've setup the backend switch for this.

**Comment by Jujhar** [comment]
Interested in fixing some of your issues when you jump back on board in this project.
codemill$35
